<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_first_name">
            <property name="text">
             <string>First Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_last_name">
            <property name="text">
             <string>Last Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_favorite_fruit">
            <property name="text">
             <string>Favorite Fruit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="line_edit_first_name"/>
          </item>
          <item>
           <widget class="QLineEdit" name="line_edit_last_name"/>
          </item>
          <item>
           <widget class="QComboBox" name="combo_box_favorite_fruit">
            <item>
             <property name="text">
              <string>Select one...</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Apples</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bananas</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Oranges</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="button_say_hello">
          <property name="text">
           <string>Say Hello</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="text_browser_output">
      <property name="placeholderText">
       <string>Message ...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>358</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionMenu_Item_1"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionMenu_Item_1">
   <property name="text">
    <string>Menu Item 1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
